src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -164,35 +164,33 @@ platforms. Just look at that hard-coded path to Chrome :(
 #### Using ChromeLauncher {: #nodechromelauncher }
 
 [Lighthouse](/web/tools/lighthouse/) is a marvelous
-tool for testing the quality of your web apps. One thing people don't realize
-is that it ships with some really nice helper modules for working with Chrome.
-One of those modules is `ChromeLauncher`. `ChromeLauncher` will find where
+tool for testing the quality of your web apps. A robust module for launching
+Chrome was developed within Lighthouse and is now extracted for standalone use.
+The [`chrome-launcher` NPM module](https://www.npmjs.com/package/chrome-launcher)
+will find where
 Chrome is installed, set up a debug instance, launch the browser, and kill it
 when your program is done. Best part is that it works cross-platform thanks to
 Node!
 
-Note: The Lighthouse team is exploring a standalone package for `ChromeLauncher` with
-an improved API. Let us know if you have [feedback](https://github.com/GoogleChrome/lighthouse/issues/2092).
-
-By default, **`ChromeLauncher` will try to launch Chrome Canary** (if it's
+By default, **`chrome-launcher` will try to launch Chrome Canary** (if it's
 installed), but you can change that to manually select which Chrome to use. To
-use it, first install Lighthouse from npm:
+use it, first install from npm:
 
-    yarn add lighthouse
+    yarn add chrome-launcher
 
-**Example** - using `ChromeLauncher` to launch Headless
+**Example** - using `chrome-launcher` to launch Headless
 
 ```javascript
-const chromeLauncher = require('lighthouse/chrome-launcher/chrome-launcher');
+const chromeLauncher = require('chrome-launcher');
 
```

Comment:
It is, but I'd like to add it here so people don't have to go reading elsewhere. You could add "// Optional: set logging for launcher".